doc/esil.md

```diff
@@ -24,6 +24,52 @@ peeks and pokes, spawning a syscall, etc.
 
     [0x00000000]> e asm.esil = true
 
+Running ESIL
+============
+
+In visual mode, `V`, one can iterate through the instructions via the `s` (step) key
+and see how registers are changing interactively as `;-- pc` (program counter) advances,
+just like in r2's debug facilities:
+
+[0x00100004 [xaDvc]0 2% 395 bin/ired_v850]> diq;?t0;f .. @ entry0+4 # 0x100004
+dead at 0x00000000
+- offset -   0 1  2 3  4 5  6 7  8 9  A B  C D  E F  0123456789ABCDEF
+0x00200000  ffff ffff ffff ffff ffff ffff ffff ffff  ................
```

Comment:
run aeim to initialize a fake stack or you'll get this ffff. but anyway, using aes and dr= should be enough for testing purposes. the visual mode is just the same as in the debugger